flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -1053,5 +1054,10 @@ public T getSerializer() {
 		public Map<String, String> toMap() {
 			return Collections.emptyMap();
 		}
+
+		@Override
+		public boolean equals(Object obj) {
+			return true;
+		}
```

Comment:
Why do we need this? If this is not needed, then please revert.

Reply:
I now understand why we need it.

Reply:
can we document the answer?

Reply:
The `equals` method is needed for the `equals` method of the `ExecutionConfig` which was not needed before because the field was `null`.